src/style/mod.rs

```diff
@@ -42,7 +47,7 @@ pub enum Display {
 
 impl Default for Display {
     fn default() -> Self {
-        Self::Flex
+        Self::None // FIXME: What should the default be if the flexbox feature is disabled?
```

Comment:
@nicoburns What is the most logical value to keep as default here? It can't be `Self::Flex` when only using taffy with the `grid` feature enabled. None is the only value that spans both algorithms.

Reply:
If grid is enabled but not flex then I think grid should be default. You can use cfg to make it conditional. You can also derive Default in recent versions of Rust. You stick `#[default]` on the variant you want to be the default (which you can control w moth cfg_attr)